simulation/chart.py

```diff
@@ -150,120 +175,119 @@ class ChartWidget:
     """
 
     def __init__(self, chart: pg.PlotItem) -> None:
-        self.__chart = chart
-        self.__plots: Dict[str, pg.PlotDataItem] = {}
+        self._chart = chart
+        self._plots: Dict[str, pg.PlotDataItem] = {}
 
     def add_line_plot(
         self,
         id: str,
         name: str,
-        color: Optional[Color] = None,
-        width: Optional[float] = None,
-        style: Optional[PenStyle] = None,
-        autoDownsample: Optional[bool] = None,
-        downsample: Optional[int] = None,
-        downsampleMethod: Optional[DownsampleMethod] = None,
+        color: Color = DEFAULT_PEN_COLOR,
```

Comment:
Is there a reason you got rid of `Optional` here?

Reply:
As you requested defaults which I've derived from base pyqtgraph calls. Optional is unnecessary due to replacement. For example, when color is None it is replaced by RGB(200,200,200) value on `mkPen` call. Thus makes hidden defaults visible to caller. I only retained `Optional` when there is a meaningful logic behind.